test/tests-fuzz.cpp

```diff
@@ -3,6 +3,8 @@
 
 #include <rapidfuzz/fuzz.hpp>
 
+#include <iostream>
+
```

Comment:
I think this is an artifact of merging in the main branch, which should be removed.

Reply:
done.